### PR TITLE
Neumorphism animation

### PR DESCRIPTION
**Pull request by kalpanaT-05**
Done creating animation and adding username in animation.js

### CODE REVIEW

File: animations/neumorphism-animation.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    animation: typing 4s steps(30) 1s 1 alternate both,
+               cursorAnimation 800ms steps(30) infinite normal;
+
+    border-right: 1px solid #eee;
+}
+
+.title{
+    font-size: 6.5em !important;
+    font-family: 'Fira Code', sans-serif;
+    font-weight: bold;
```
Comment (NiallEccles) [inline animations/neumorphism-animation.css:20]: Could you move these styles into your `.neumorphism-animation` class?

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Thanks for submitting this pull request! I just have one small change for you to make before i can merge this.

File: animations/neumorphism-animation.css (hunk 1 above)
Comment (kalpanaT-05) [reply animations/neumorphism-animation.css:20]: okay sure i will make it

Comment (kalpanaT-05) [reply animations/neumorphism-animation.css]: done

**Comment by kalpanaT-05** [comment]
thank you so much @NiallEccles :)

**Comment by NiallEccles** [comment]
@kalpanaT-05 no problem!

**Pull request merged**